### PR TITLE
Add build script for Hwloc

### PR DESCRIPTION
**Pull request by mbauman**
Super simple, `./configure ...; make; make install` worked for all architectures on the first try.

### CODE REVIEW

File: Hwloc/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    Linux(:powerpc64le, libc=:glibc),
+    Linux(:i686, libc=:musl),
+    Linux(:x86_64, libc=:musl),
+    Linux(:aarch64, libc=:musl),
+    Linux(:armv7l, libc=:musl, call_abi=:eabihf),
+    MacOS(:x86_64),
+    FreeBSD(:x86_64),
+    Windows(:i686),
+    Windows(:x86_64)
+]
```
Comment (staticfloat) [inline Hwloc/build_tarballs.jl]: Best to just use `platforms = supported_platforms()` here so that it automatically tries new ones when they get included.

**Comment by mbauman** [comment]
Great, thanks! I missed the requested change notification — sorry! Looks like you fixed that up yourself.

File: Hwloc/build_tarballs.jl (hunk 1 above)
Comment (mbauman) [reply Hwloc/build_tarballs.jl]: Perhaps the wizard should automatically do this in cases where all platforms worked?

Comment (staticfloat) [reply Hwloc/build_tarballs.jl]: You are correct.

**Pull request merged**